src/ipyradiant/__init__.py

```diff
@@ -4,6 +4,21 @@
 from ._version import __version__
 from .loader import LoadWidget
 from .query import QueryWidget
-from .visualization import CytoscapeVisualization
+from .visualization import (
+    CytoscapeVisualization,
+    DatashaderVis,
+    NXBase,
```

Comment:
If base classes are not supposed to be directly used by users (and are just building blocks for actual widget classes) we probably don't need to import them to the top level of the package.